modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -203,47 +203,73 @@ Each provider has its own configuration map that can be passed to the `genai.vec
 | model
 | STRING
 | The name of the model you want to invoke. +
-Supported values: `"textembedding-gecko@001"`
+ +
+Supported values: +
+ +
+
+label:new[Introduced in 5.17]:
```

Comment:
I don't think the colons work well when rendered

Reply:
should we not mention that open ai _used_ to be restricted until 5.18?

Reply:
> I don't think the colons work well when rendered

I'm not super happy about it either but I'm not sure it's better without it.

> should we not mention that open ai used to be restricted until 5.18 (or in 5.17)

Hmm, probably, yeah. On the other hand I'm not sure if this table should be a full changelog / reflect all versions...